core/src/price_estimation.rs

```diff
@@ -48,6 +48,40 @@ pub struct PriceOracle {
     source: Box<dyn PriceSource + Send + Sync>,
 }
 
+pub fn default_price_source(
```

Comment:
nit: Should this method live in the price_source module directly?

Reply:
The PriceOracle is what we use to give prices to the solver and this set up of average and priority is fairly specific to it so it makes sense here.

Reply:
Right, but if we want to reuse this setup in the price-estimator it's no longer specific to the PriceOracle but more to the PriceSource trait in general, or am I missing something?

Reply:
In my mind, which price sources are used is decided by the price oracle. The price estimator just copies that. Looking at the modules the price source files just defines the trait and the implementations of it live in files on the same level. The price_estimator file is one level higher and uses them all. So that file is really the one bringing it all together.
It's not a big thing, just felt a little more appropriate that way.